index.js

```diff
@@ -1,30 +1,47 @@
 function playRound(playerSelection, computerSelection) {
   playerSelection = playerSelection.toLowerCase();
   computerSelection = computerSelection.toLowerCase();
-  if (playerSelection === "rock") {
-    if (computerSelection === "rock") {
-      return "None";
-    } else if (computerSelection === "papper") {
-      return "Lose";
-    } else {
-      return "Win";
-    }
-  } else if (playerSelection === "papper") {
-    if (computerSelection === "papper") {
-      return "None";
-    } else if (computerSelection === "rock") {
-      return "Win";
-    } else {
-      return "Lose";
-    }
-  } else {
-    if (computerSelection === "papper") {
-      return "Win";
-    } else if (computerSelection === "rock") {
-      return "Lose";
-    } else {
-      return "None";
-    }
+  switch (playerSelection) {
+    case "rock":
+      switch (computerSelection) {
+        case "rock": {
+          return "None";
+        }
```

Comment:
small nit: you don't need braces for single-line cases.

`case "rock": return "None";`